### PR TITLE
fix(create secret): create secret using backend endpoint

### PR DESCRIPTION
**Pull request by elysee15**
Capture d’écran 2021-08-05 à 20 10 44

### CODE REVIEW

File: src/components/CreateSecretForm.tsx [hunk 1]
```diff
@@ -43,46 +46,61 @@ const initialValues = {
...
+		};
+
+		createSecret(data).then(
+			(response: AxiosResponse) => {
+				setToken(response.data);
+
+				helpers.setSubmitting(false);
+			},
+			(error: AxiosError) => {
+				console.error("[error when creating secret]", error.message);
```
Comment (bbengfort) [inline src/components/CreateSecretForm.tsx]: We'll want some user-visible error messages eventually, but this is a good start!

File: src/utils/interfaces/Lifetime.ts [hunk 2]
```diff
@@ -1,4 +1,6 @@
+export type Value = "5m" | "15m" | "30m" | "1h" | "2h" | "3h" | "24h" | "48h" | "72h" | "168h";
```
Comment (bbengfort) [inline src/utils/interfaces/Lifetime.ts]: Nice!

**Review by bbengfort — APPROVED** [review]
Excellent!

File: src/components/CreateSecretForm.tsx (hunk 1 above)
Comment (elysee15) [reply src/components/CreateSecretForm.tsx]: Okay

**Pull request merged**